docs/conf.py

```diff
@@ -272,6 +283,19 @@
     "term": "tooltip",
 }
 
+# Using sphinx.ext.extlinks lets us simplify the process of creating
+# links to commonly used external sites. The key of the extlink becomes
+# a new role, and the corresponding tuple contains the base url and the
+# caption. For example, we can now do :orcid:`0000-0000-0000-0000` and
+# have a link create to the corresponding ORCID page. New roles should
+# be added to rst-roles in setup.cfg to avoid being caught by
+# flake8-rst-docstrings.
+
+extlinks = {
+    "orcid": ("https://orcid.org/%s", "%s"),
+    "wikipedia": ("https://en.wikipedia.org/wiki/", "%s"),
```

Comment:
An alternative for this would be [sphinx-ext-wikipedia](https://github.com/quiver/sphinx-ext-wikipedia).  However, it hasn't been released on PyPI or updated since 2013.